### ISSUE TITLE
Commit 1ef9039bcc breaks build on NetBSD

### ISSUE THREAD

**Issue by jkeenan** [Perl/perl5#19909]
Commit 1ef9039bcc appears to have broken the build on NetBSD.

I am inferring this from examination of NetBSD smoke-test results at [perl5.test-smoke.org](https://perl5.test-smoke.org/search?selected_osnm=netbsd&report_osnm=freebsd&selected_perl=all).  Note that blead was PASSing at `v5.37.1-39-g59ead93c94`, but that a branch at `v5.37.1-41-gad1bb8a058` -- only 2 commits later -- is FAILing during `make`.  The branch in question, smoke-me/tonyc/19894-skip-unicode-dollarzero, was forked from blead at the commit in-between the two, which was this:
```
$ git describe
v5.37.1-40-g1ef9039bcc

$ gitshowf
commit 1ef9039bccbfe64f47f201b6cfb7d6d23e0b08a7 (HEAD -> blead, origin/blead, origin/HEAD)
Author:     Karl Williamson <khw@cpan.org>
AuthorDate: Tue Jun 21 11:45:06 2022
Commit:     Karl Williamson <khw@cpan.org>
CommitDate: Wed Jun 29 17:22:51 2022

    Eval param to SvPV-foo exactly once
    
    This changes all the API macros that retrieve a PV into a call to an
    inline function so as to evaluate the parameter just once.
```
The string `sv_pvutf8n_force_wrapper` occurs in the `git diff` output for the commit above.  It also occurs in the smoke-test failure logs, e.g., https://perl5.test-smoke.org/report/5019838.

@khwilliamson, can you take a look?

@cjg-cguevara (and anyone else with access to NetBSD), can you confirm that the build breaks at 1ef9039bccbfe64f47f201b6cfb7d6d23e0b08a7?

Thank you very much.
Jim Keenan

**Comment by jkeenan** [comment]
This branch is failing somewhat similarly on NetBSD:
http://perl.develop-help.com/?b=smoke-me%2Fkhw-fix_inline

**Comment by jkeenan** [comment]
With additional commits, the [smoke-me/khw-fix_inline](http://perl.develop-help.com/?b=smoke-me%2Fkhw-fix_inline) branch now is starting to PASS.   Will continue to monitor.

**Comment by khwilliamson** [comment]
Fixed by 409173239aadc465992374048dfa7b54441ce82e
